README.md

```diff
@@ -137,10 +142,6 @@ EXAMPLES
 
     kvm-install-vm create -T UTC foo
         Create a default VM with UTC timezone.
-
-    kvm-install-vm create -s ~/script.sh -g vnc -u bar foo
```

Comment:
Does this example not work anymore?

Reply:
It still seems to work. As far as I can tell this only ever existed in the readme and not in the commands help. (https://github.com/giovtorres/kvm-install-vm/commit/8670857164eb0e78e1707c1a9fd8d65132ef0bfd seems to be when it was first added to the readme, can't find any references to it in the main script.)

I can re-add it all if that's preferred?